plat/rockchip/common/aarch64/plat_helpers.S

```diff
@@ -192,20 +195,23 @@ endfunc plat_crash_console_putc
 	.align	16
 func platform_cpu_warmboot
 	mrs	x0, MPIDR_EL1
-	and	x1, x0, #MPIDR_CPU_MASK
-	and	x0, x0, #MPIDR_CLUSTER_MASK
+	and	x19, x0, #MPIDR_CPU_MASK
+	and	x20, x0, #MPIDR_CLUSTER_MASK
+#if NEED_PMU_MACROS_S
+	func_rockchip_clst_warmboot
+#endif
 	/* --------------------------------------------------------------------
 	 * big cluster id is 1
 	 * big cores id is from 0-3, little cores id 4-7
 	 * --------------------------------------------------------------------
 	 */
-	add	x0, x1, x0, lsr #6
+	add	x21, x19, x20, lsr #PLATFORM_CLST_TO_CPUID_SHIFT
```

Comment:
Maybe the value keeping x21 will be used as a parameter with invoking function like x20 in func_rockchip_clst_warmboot。